src/Expense.java

```diff
@@ -1,7 +1,32 @@
+import java.util.Scanner;
 
 public class Expense {
+	Scanner scnr = new Scanner(System.in);
 	String source;
 	double amount;
 	int yearlyfrequency; //1 for 1 time or once a year, 12 for monthly or or 24 for biweekly
-	//should add contructor(s)
+	
+	public String addSource() {

```

Comment:
I'd recommend sticking with getter / setter convention where possible as that is considered good practice.  Tweak names from `addAmount` to `setAmount` since this class will be used in the User class as an object in its `ArrayList<Expense>` area.  

An example of how I would implement this and removing the Scanner objects would be:
```Java
public void setSource(String source) {
		this.source = source;
	}
```